### ISSUE TITLE
RCS1163 Unused parameter '_'

### ISSUE THREAD

**Issue by wanton7** [JosefPihrt/Roslynator#235]
Roslynator incorrectly reports _ (C# 7 parameter ignore) as unused parameter.

```csharp
ThreadPool.UnsafeQueueUserWorkItem(_ =>
{
}, null);
```

**Comment by wanton7** [comment]
Seems that this is actually not supported yet by the language. https://github.com/dotnet/csharplang/issues/111
